bindgen/BUILD.bazel

```diff
@@ -19,10 +19,13 @@ rust_bindgen_toolchain(
         "//conditions:default": "@bindgen_clang_linux//:clang",
     }),
     libclang = select({
-        "//rust/platform:osx": "@bindgen_clang_osx//:libclang.so",
-        "//conditions:default": "@bindgen_clang_linux//:libclang.so",
+        "//rust/platform:osx": "@bindgen_clang_osx//:libclang",
+        "//conditions:default": "@bindgen_clang_linux//:libclang",
+    }),
+    libstdcxx = select({
+        "//rust/platform:osx": "@bindgen_clang_osx//:libc++",
+        "//conditions:default": None,
```

Comment:
Could this perhaps be the reason the build succeeds on linux?

Reply:
No, this attribute does not affect the rpath in the resulting binary